api/src/wfl/once.clj

```diff
@@ -1,19 +1,43 @@
 (ns wfl.once
-  "Manage credentials."
+  "Manage credentials and configurations."
   (:require [clojure.data.json :as json]
             [clojure.java.io   :as io]
             [clojure.string    :as str]
+            [clojure.tools.logging :as log]
             [wfl.environments :as env]
+            [vault.client.http]         ; vault.core needs this
+            [vault.core :as vault]
             [wfl.util         :as util])
   (:import [com.google.auth.oauth2 GoogleCredentials]))
 
+;; Only load the relevant environment variables
+;;
 (defonce the-system-environments
-  (reduce #(assoc %1 %2 (System/getenv %2)) {}
-          ["COOKIE_SECRET"
-           "CROMWELL"
-           "GOOGLE_APPLICATION_CREDENTIALS"
-           "WFL_OAUTH2_CLIENT_ID"
-           "WFL_DEPLOY_ENVIRONMENT"]))
+  (delay (reduce #(assoc %1 %2 (System/getenv %2)) {}
+           ["COOKIE_SECRET"
+            "CROMWELL"
+            "GOOGLE_APPLICATION_CREDENTIALS"
+            "USER"
+            "VAULT_TOKEN"
+            "WFL_OAUTH2_CLIENT_ID"
+            "WFL_POSTGRES_URL"
+            "WFL_POSTGRES_USERNAME"
+            "WFL_POSTGRES_PASSWORD"
+            "WFL_DEPLOY_ENVIRONMENT"])))
```

Comment:
Hence we don't need to document them in markdown. Downside is this loses a bit of flexibility (but do we really need that kind of flexibility? i.e. load env vars that defined in runtime dynamically)